radixdlt/src/main/java/com/radixdlt/middleware2/network/MessageCentralValidatorSync.java

```diff
@@ -165,8 +174,17 @@ public void sendGetVerticesErrorResponse(BFTNode node, HighQC highQC) {
 					return null;
 				}
 
-				BFTNode node = BFTNode.create(src.getSystem().getKey());
-				return new GetVerticesErrorResponse(node, msg.highQC());
+				if (errorResponseRateLimiter.tryAcquire()) {
+					BFTNode node = BFTNode.create(src.getSystem().getKey());
+					return new GetVerticesErrorResponse(node, msg.highQC());
+				} else {
+					var counter = counters.add(CounterType.NETWORKING_DROPPED_ERROR_RESPONSES, 1);
+
+					if (counter % 1000 == 0) {
```

Comment:
Would be better as `== 1` so we don't have to wait for 1000 messages to be dropped before seeing the warning.

Reply:
Added logging of first messages.